### ISSUE TITLE
lint/switchif: handle switch statements with multiple "case" values properly

### ISSUE THREAD

**Issue by Quasilyte** [go-critic/go-critic#180]
```
github.com/go-critic/go-critic/lint/stddef_checker.go:192:4: switchif: should rewrite switch statement to if statement
```
```go
switch fn.Name {
case "uint", "int":
	c.warn(call, "math/bits.UintSize")
}
```
This switch has 1 case, but that case has 2 values.
It has the same semantics as:
```go
switch fn.Name {
case "uint":
	c.warn(call, "math/bits.UintSize")
case "int":
	c.warn(call, "math/bits.UintSize")
}
// Or 
switch fn.Name {
case "uint":
	fallthrough
case "int":
	c.warn(call, "math/bits.UintSize")
}
```
`switchif` should not give a warning in this case.

**Comment by Quasilyte** [comment]
@weeellz, are you willing to solve this?
It leads to false positives.

**Comment by cristaloleg** [comment]
Can we just ignore cases with multiple values? 🤔

**Comment by Quasilyte** [comment]
For this checker, yes. 
If there are multiple cases, this should be interpreted as "don't care".
This is like 1 additional check before generating a warning.

**Comment by cristaloleg** [comment]
At first I though we should rewrite a switch like in 2 last examples. Misunderstood.

**Comment by weeellz** [comment]
@Quasilyte, how to reproduce this warning?

**Comment by Quasilyte** [comment]
@weeellz, this switch does trigger a warning:
```go
switch x {
case 1, 2: // Has two values, but triggers warning
}
```

**Comment by weeellz** [comment]
I mean how to run linter to get this:
`github.com/go-critic/go-critic/lint/stddef_checker.go:192:4: switchif: should rewrite switch statement to if statement`

**Comment by Quasilyte** [comment]
This should work:
`gocritic check-package -enable=switchif github.com/go-critic/go-critic/lint`

**Comment by weeellz** [comment]
And how could I get/count values from case?

**Comment by Quasilyte** [comment]
It's `CaseClause.List`.
nil - default, otherwise it's slice of case expressions.
